src/rules/root-no-standard-properties/index.js

```diff
@@ -19,6 +19,12 @@ export default function (actual) {
     root.walkRules(rule => {
       if (rule.selector.indexOf(":root") === -1) { return }
 
+      // ignore :not(:root) selectors
+      for (const selector of rule.selectors) {
+        const notSelectorWithRoot = selector.indexOf(":not") !== -1 && selector.indexOf(":root") > 0
```

Comment:
These means of identification doesn't seem precise enough to me. Wouldn't this allow the following to pass, and shouldn't: `.foo, :root, .bar:not(:hover) {}`?